lib/modules/manager/cargo/index.ts

```diff
@@ -3,17 +3,17 @@ import { CrateDatasource } from '../../datasource/crate';
 import * as cargoVersioning from '../../versioning/cargo';
 import { updateArtifacts } from './artifacts';
 import { extractPackageFile } from './extract';
+import { getRangeStrategy } from './range';
```

Comment:
```suggestion
export { getRangeStrategy } from './range';
```

i would prefer direct export, as it's not used in this file